index.html

```diff
@@ -6,65 +6,65 @@
     <link
       rel="apple-touch-icon"
       sizes="180x180"
-      href="/favicons/polkadot/apple-touch-icon.png"
+      href="/favicons/creditcoin/apple-touch-icon.png"
     />
     <link
       rel="icon"
       type="image/png"
       sizes="32x32"
-      href="/favicons/polkadot/favicon-32x32.png"
+      href="/favicons/creditcoin/favicon-32x32.png"
     />
     <link
       rel="icon"
       type="image/png"
       sizes="16x16"
-      href="/favicons/polkadot/favicon-16x16.png"
+      href="/favicons/creditcoin/favicon-16x16.png"
     />
     <link
       rel="mask-icon"
-      href="/favicons/polkadot/safari-pinned-tab.svg"
+      href="/favicons/creditcoin/safari-pinned-tab.svg"
       color="#e6007a"
     />
     <meta name="msapplication-TileColor" content="#e6007a" />
     <meta name="theme-color" content="#fff" />
 
     <meta
       name="title"
-      content="Polkadot Staking Dashboard | Polkadot Staking (DOT)"
+      content="Creditcoin Staking Dashboard | Creditcoin Staking (CTC)"
     />
     <meta
       name="description"
-      content="Polkadot Staking Dashboard is the easiest way to stake DOT, check validator stats, manage your nominations and join nomination pools. Stake on Polkadot (DOT)."
+      content="Creditcoin Staking Dashboard is the easiest way to stake CTC, check validator stats, manage your nominations and join nomination pools (Coming Soon). Stake on Creditcoin (CTC)."
```

Comment:
We already added support for nomination pools in https://github.com/gluwa/creditcoin/pull/1074/, right?